tests/e2e/helpers/keys.ts

```diff
@@ -42,22 +43,104 @@ export const keyTypes = [
  * @param count The count of keys to add
  * @param keyNameStartWith The name of the key
  */
-export async function populateDBWithHashes(host: string, port: string, count: number, keyNameStartWith: string): Promise<void> {
+export async function populateDBWithHashes(host: string, port: string, keyArguments: AddKeyArguments): Promise<void> {
```

Comment:
Remove `count` from the description since it's not relevant anymore

Reply:
fixed